tests/multipleBackend/routes/routeBackbeat.js

```diff
@@ -1529,6 +1537,9 @@ describe('backbeat routes', () => {
                                 TEST_ENCRYPTED_BUCKET : TEST_BUCKET,
                             objectKey: testCase.encodedKey,
                             resourceType: 'metadata',
+                            queryObj: {
+                                versionId: versionIdUtils.encode(testMd.versionId),
+                            },
                             authCredentials: backbeatAuthCredentials,
```

Comment:
what is this extra parameter?
this was not required before (at least in Zenko), why do we need to pass it?

Here, it seems the tests are about creating a new version automatically, so the caller (practically, that would be backbeat) should not provide a versionID ;
If we have another feature where we can create an object with a specific VersionID, that should be a separate test (if not already implemented)

Reply:
This is not required, I removed it.

It came from branch 7.70 where the test did not create new version but updated the same versionId